core/embed/rust/src/ui/model_tt/component/result.rs

```diff
@@ -130,13 +130,13 @@ impl<'a, T: StringType> ResultScreen<'a, T> {
     }
 }
 
-impl<'a, T: StringType> Component for ResultScreen<'a, T> {
+impl<'a> Component for ResultScreen<'a> {
     type Msg = Never;
 
     fn place(&mut self, _bounds: Rect) -> Rect {
         self.bg.place(screen());
 
-        let (main_area, footer_area) = ResultFooter::<&'a str>::split_bounds();
+        let (main_area, footer_area) = ResultFooter::<'a>::split_bounds();
```

Comment:
...so that you don't have to specify the lifetime here (avoiding the need to `impl<'a> Component`)...

Reply:
d0243af2e2ecd705cf927cfa3220d61d107fc09b